### PR TITLE
Update GetMemberBy Route

### LINKED ISSUE #61: Get multiple users
As the backends now allows to retrieve multiple users at once, some of the routes have changed.
Query client needs to be updated

- member?email= is now member/search?email=
- added member?id=x&id=y to retrieve multiple users at once

### PR DESCRIPTION
**Pull request by Julien-Torrent**
Updates query-client for [graasp#73](https://github.com/graasp/graasp/issues/73)

close #61

### CODE REVIEW

**Review by juancarlosfarah — APPROVED** [review]
👍

File: src/api/routes.ts [hunk 1]
```diff
@@ -46,8 +46,10 @@ export const buildPostItemChatMessageRoute = (id: UUID) =>
   `${ITEMS_ROUTE}/${id}/chat`;
 
 export const buildGetMemberBy = (email: string) =>
-  `${MEMBERS_ROUTE}?email=${email}`;
+  `${MEMBERS_ROUTE}/search?email=${email}`;
```
Comment (pyphilia) [inline src/api/routes.ts:49]: Does this route exist? Did we add `search` in the route in `graasp`?

**Review by pyphilia — APPROVED** [review]
It looks good to me. I just have a question regarding the new `get member by` route.

File: src/api/routes.ts (hunk 1 above)
Comment (Julien-Torrent) [reply src/api/routes.ts:49]: this comes from https://github.com/graasp/graasp/pull/76

**Pull request merged**